Tests/OrdersTests/OrdersTests.swift

```diff
@@ -1,15 +1,27 @@
 import XCTVapor
+import Fluent
+import FluentSQLiteDriver
 @testable import Orders
 
 final class OrdersTests: XCTestCase {
-    func testExample() {
-        // This is an example of a functional test case.
-        // Use XCTAssert and related functions to verify your tests produce the correct
-        // results.
-        //XCTAssertEqual(OrdersService().text, "Hello, World!")
-    }
+    var app: Application!
+    let orderDelegate = OrderDelegate()
+    
+    override func setUp() async throws {
+        self.app = try await Application.make(.testing)
+
+        app.databases.use(.sqlite(.memory), as: .sqlite)
+        OrdersService.register(migrations: app.migrations)
+        app.migrations.add(CreateOrderData())
+        let ordersService = try OrdersService(app: app, delegate: orderDelegate)
+        app.databases.middleware.use(OrderDataMiddleware(service: ordersService), on: .sqlite)
 
-    static var allTests = [
-        ("testExample", testExample),
-    ]
+        try await app.autoMigrate()
+    }
+    
+    override func tearDown() async throws { 
+        try await app.autoRevert()
+        try await self.app.asyncShutdown()
+        self.app = nil
```

Comment:
I don't think this is needed

Reply:
I had removed it, but I had to add it back to cover the `revert` method in testing